### PR TITLE
Add documentation for cloud setup token

### PR DESCRIPTION
**Pull request by mmanes**
FusionAuth Cloud deployments now require a setup token to access the Setup Wizard. 

- Update screenshots and document this process.
- Cleanup fa-screenshot.sh

Resolves PLA-91

### CODE REVIEW

File: src/fa-screenshot.sh [hunk 1]
```diff
@@ -190,14 +171,10 @@ printOut "-- Checking image properties"
 printFileAttribs
 
 printOut "-- Resizing screenshot"
-convert=`which convert`
-$convert -geometry 1600x ${absFile} ${absFile}
-
-printOut "-- Compressing png"
-compressUsingTinyPNG
+magick=`which magick`
```
Comment (bhalsey) [inline src/fa-screenshot.sh:174]: If magick is in the path, why bother fully qualifying it?

File: src/fa-screenshot.sh [hunk 2]
```diff
@@ -190,14 +171,10 @@ printOut "-- Checking image properties"
...
+$magick ${absFile} -geometry 1600x ${absFile}
```
Comment (bhalsey) [inline src/fa-screenshot.sh:175]:
```suggestion
$magick convert ${absFile} -geometry 1600x ${absFile}
```
pretty sure we need the tool name in here

**Review by bhalsey — CHANGES_REQUESTED** [review]
We should also remove references to tinyPNG in `DocsDevREADME.md`

File: src/fa-screenshot.sh (hunk 1 above)
Comment (mooreds) [reply src/fa-screenshot.sh:174]: Note that the github action shrink images uses tinypng right now too. https://github.com/FusionAuth/fusionauth-site/blob/main/.github/workflows/shrink-images.yml as does the `shrink-images.sh` script.

Comment (mmanes) [reply src/fa-screenshot.sh:174]: Yeah, the `which` bit seems redundant.

File: src/fa-screenshot.sh (hunk 2 above)
Comment (mmanes) [reply src/fa-screenshot.sh:175]: I thought so too, but it complains if you do.

**Pull request merged**